content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -99,44 +99,39 @@ marked as a `metrics` file with `-M`. Its contents are:
 }
 ```
 
-> DVC doesn't force you to use any specific file names, nor does it enforce a
-> format or structure of a metrics file. It's completely user/case-defined.
-> Refer to `dvc metrics` for more details.
-
-You can view tracked metrics with DVC:
+You can view tracked metrics with `dvc metrics show `:
 
 ```dvc
 $ dvc metrics show
 Path                    avg_prec.test    avg_prec.train    roc_auc.test    roc_auc.train
 eval/live/metrics.json  0.94496          0.97723           0.96191         0.98737
 ```
 
-## Configuring plots
+[`evaluate.py`]:
+  https://github.com/iterative/example-get-started/blob/master/src/evaluate.py
+[roc-auc]:
+  https://scikit-learn.org/stable/modules/model_evaluation.html#receiver-operating-characteristic-roc
+[average precision]:
+  https://scikit-learn.org/stable/modules/model_evaluation.html#precision-recall-and-f-measures
+[metrics file]: /doc/command-reference/metrics#supported-file-formats
+
+The stage also writes [`roc_curve`] and [`confusion_matrix`] values in the
+`eval/live/plots` directory. Let's see how to visualize these better...
 
-`evaluate.py` also writes `precision`, `recall`, and `thresholds` arrays
-(obtained using
-[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html))
-into the plots files `eval/prc/train.json` and `eval/prc/test.json`:
+## Rendering plots
+
+Similarly, `evaluate` writes `precision`, `recall`, and `thresholds` values into
+JSON arrays (shown below) in the the `eval/prc/train.json` and
+`eval/prc/test.json` [plots files]. It also generates a custom
+`eval/importance.png` image showing a bar chart of features' importance.
 
 ```json
-{
-  "prc": [
-    { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
-    ...,
-    { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.6 }
-  ]
-}
+{ "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
+...
 ```
 
-Similarly, it writes arrays for the
-[roc_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html)
-and
-[confusion matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
-into JSON files in the `eval/live/plots` directory, and an image
-`eval/importance.png` with a feature importance bar chart for additional plots.
-
-To view plots, first configure the axes and other specifications of your plots
-by adding a `plots` section to your `dvc.yaml`:
+You can visualize these different elements as plots with DVC! But first, you
+must configure their [rendering properties][plots files]:
```

Comment:
Do we need to clarify that this is part of `dvc.yaml`? Why introduce the term "rendering properties"?

More generally (but minor), why so much use of "rendering" in this PR? I don't have strong opinion on it, but I'm not sure it's more clear than what it's replacing.

Reply:
Changed to "configure the plots in dvc.yaml". Changed other instances of "render" to "visualizing" and "generate", for consistency with existing docs terminology (good catch).